sklearn/covariance/_empirical_covariance.py

```diff
@@ -279,7 +279,8 @@ def error_norm(self, comp_cov, norm='frobenius', scaling=True,
         if norm == "frobenius":
             squared_norm = np.sum(error ** 2)
         elif norm == "spectral":
-            squared_norm = np.amax(linalg.svdvals(np.dot(error.T, error)))
+            squared_norm = np.amax(linalg.svdvals(np.dot(error.T, error),
+                                                  check_finite=False,))
```

Comment:
`comp_cov` is a parameter that is not checked, so I think we should check_finite here. (or add a check when computing error ?)